projects/systelab-components/src/lib/spinner/spinner.component.ts

```diff
@@ -14,7 +14,7 @@ export class TouchspinComponent {
 	@Input() public disabled = false;
 	private _spinValues: TouchSpinValues;
 	protected _valueStr: string;
-	@Input() fillUnitsWithZero = false;
+	@Input() fillUnitsWithZero:boolean|number = false; //0 if false, and 1 if true
```

Comment:
Document the new Input in the README (https://github.com/systelab/systelab-components/tree/master/projects/systelab-components/src/lib/spinner)

Reply:
done